test/app-tap/CMakeLists.txt

```diff
@@ -6,3 +6,86 @@ target_link_libraries(module_api msgpuck)
 set(CMAKE_C_FLAGS "-Wall -Wextra -std=c99")
 set(CMAKE_C_FLAGS_DEBUG "-g -O0 -Werror")
 set(CMAKE_C_FLAGS_RELWITHDEBINFO "-O2")
+
+tarantool_make_lua_path(LUA_PATH
+  PATHS
+    ${CMAKE_CURRENT_SOURCE_DIR}/?.lua
+    ${CMAKE_CURRENT_SOURCE_DIR}/lua/?.lua
```

Comment:
Please add a comment what auto-generated file should be placed in those directories.

Reply:
My bad, it's about regular Lua files located in this directories (misread as BINARY directory at first).